src/amp-story-player/amp-story-player-impl.js

```diff
@@ -736,59 +736,64 @@ export class AmpStoryPlayer {
   }
 
   /**
-   * Shows the story provided by the URL in the player.
+   * Shows the story provided by the URL in the player and go to the page if provided.
    * @param {string} storyUrl
```

Comment:
nit: `!string` if we want to accept null.

Reply:
Well no that'd be `?string` 😅

Reply:
Haha good catch